pycbc/workflow/datafind.py

```diff
@@ -133,6 +133,19 @@ def setup_datafind_workflow(workflow, scienceSegs,  outputDir, segFilesList,
             setup_datafind_runtime_frames_single_call_perifo(cp, scienceSegs,
                                                             outputDir, tag=tag)
 
+    elif datafindMethod == "FROM_PREGENERATED_LCF_FILES":
+        ifos = scienceSegs.keys()
+        datafindcaches, datafindouts = \
+            setup_datafind_from_pregenerated_lcf_files(cp, ifos,
+                                                       outputDir, tag=tag)
+    else:
+        msg = "Entry datafind-method in [workflow-datafind] does not have "
+        msg += "expected value. Valid values are "
+        msg += "AT_RUNTIME_MULTIPLE_FRAMES, AT_RUNTIME_SINGLE_FRAMES "
+        msg += "AT_RUNTIME_MULTIPLE_CACHES or AT_RUNTIME_SINGLE_CACHES. "
+        msg += "Consult the documentation for more info."
+        raise ValueError(msg)
+
```

Comment:
It looks like you not only encountered #237, you fixed it .... but didn't tell anyone! :-\. Please let do let us know about issues like this so they can get fixed on master. The PyCBC data checking code also uses the CACHES route through the datafind module so I also hit this issue this morning.